model/src/test/java/uk/gov/hmcts/fees2/register/data/util/FeesDateUtilTest.java

```diff
@@ -27,7 +27,7 @@ public void setUp() throws ParseException {
     @Test
     public void testAddPreviousDateEODTime() {
         final Date date = FeesDateUtil.addPreviousDateEODTime(this.date);
-        Assert.assertEquals("Sat Jan 09 23:59:59 UTC 2021", date.toString());
+        //Assert.assertEquals("Sat Jan 09 23:59:59 UTC 2021", date.toString());
```

Comment:
Is this test doing anything now? With the assert commented out should the test not be removed completely?